resources/function_help/json/make_valid

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "make_valid",
+  "type": "function",
+  "groups": ["GeometryGroup"],
+  "description": "Returns a valid geometry or an empty geometry if the geometry could not be made valid.",
+  "arguments": [{
+    "arg": "geometry",
+    "description": "a geometry"
+  }],
+  "examples": [{
+    "expression": "make_valid(geom_from_wkt('POLYGON((3 2, 4 1, 5 8, 3 2, 4 2))'))",
+    "returns": "<QgsGeometry: GeometryCollection>"
```

Comment:
Is this output meaningful enough to end users? I do not recall if we have any example outputting this. What about an example that would return the actual fixed geometry as wkt? But if we keep the current example, should it be `<QgsGeometry: GeometryCollection>` or `<geometry: GeometryCollection>` or `<geometry: Polygon>`

Reply:
I don't mind adding it as a wkt, I just wanted to show the basic output. Do you think it would be better to show the resulting geometry (polygon + polyline) ? I was afraid that showing this to user might confuse them as it shows a single polygon becoming a polygon and a line.
The goes is mostly to show that the geometry type changed to something that is now 'valid'.

Reply:
Browsing geometry functions in Expression dialog, examples are usually written to output wkt, and when they should output some \<geometry:xxx\> it is described as `a xxx geometry object`, not a Qgs thing. Personally I'm fine with the polygon + polyline wkt output _(and I thought it would have output a polygon and removed the last point. Not create a line to it. So could be also beneficial for others to show it...)_